pkgs/by-name/ga/gam/package.nix

```
@@ -1,21 +1,26 @@
-{
-  lib,
-  fetchFromGitHub,
-  python3,
+{ lib
+, fetchFromGitHub
+, python3
+, yubikey-manager
+, gitUpdater
+, cacert
+,
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "gam";
-  version = "6.58";
-  format = "other";
+  version = "7.03.04";
+  format = "setuptools";
 
   src = fetchFromGitHub {
     owner = "GAM-team";
     repo = "GAM";
     rev = "refs/tags/v${version}";
```

```suggestion
    tag = "v${version}";
```
This is a new optional argument for `fetchFromGitHub`, see cb9f9a1e5a51a87d59b373db7016cee1608debd5.